app/views/shared/_header.html.erb

```diff
@@ -27,7 +27,14 @@
       <%= render partial: 'shared/navbar_search', locals: { search_form_id: 'dm-navbar-search-desktop-form', search_field_id: 'dm-navbar-search-desktop-field', search_button_id: 'dm-navbar-search-desktop-button' } %>
       <%= render partial: 'shared/header_user_login_nav', locals: { accordion_id: 'dm-user-profile-options' } %>
     </div>
-    <button class="usa-menu-btn bg-ink padding-x-205">Menu</button>
+    <button class="usa-menu-btn" id="originalMenuButton"></button>
+    <i
+      class="fa fa-bars padding-x-3 desktop:display-none"
+      aria-label="Menu"
+      id="customMenuButton"
+      tabindex="0"
+      role="button"
+    ></i>
```

Comment:
Good instinct to add an `aria-label` after removing the `Menu` text. But leaving the old button markup there feels a little crufty. Nesting the icon inside of the `<button>` tag will get us the keyboard nav behavior for free:
```suggestion
    <button class="usa-menu-btn" id="originalMenuButton">
    	<i class="fa fa-bars" aria-label="menu"></i>
    </button>
```

Reply:
oh nice! that's a lot cleaner thank you.